libs/ledger-live-common/src/generated/account.ts

```diff
@@ -18,10 +16,8 @@ export default {
   cosmos,
   crypto_org,
   elrond,
-  filecoin,
-  internet_computer,
   near,
-  stacks,
+  tron,
```

Comment:
any clue why coins disappearing from here?

Reply:
Yes, these are coins that had some unneeded de/serialization, i.e their `extra` only contains serializable types, so I could just remove it.